backend/pipelined/filter.go

```diff
@@ -30,22 +30,28 @@ func evaluateEventFilterStatement(event *types.Event, statement string) bool {
 	return result
 }
 
-// Returns true if the event should be filtered.
+// Returns true if the event should be filtered/denied.
 func evaluateEventFilter(event *types.Event, filter *types.EventFilter) bool {
+	fields := utillogging.EventFields(event, false)
+	fields["filter"] = filter.Name
+	fields["check"] = event.Check.Name
+	fields["entity_name"] = event.Entity.ID
```

Comment:
The entity ID and check are already provided by utillogging.EventFields. We may wish to amend the EventFields function to use the check name instead of the entire check.